README.md

```diff
@@ -207,6 +207,8 @@ assertTextColorIsNot(R.id.someColorListText, R.color.another_state_list);
 
 // Check recyclerView item count against expected item count
 assertRecyclerViewItemCount(R.id.recycler, 10);
+assertDisplayedAtPosition(R.id.recycler, 0, "text");
+assertDisplayedAtPosition(R.id.recycler, 0, R.id.text_field, "text");
```

Comment:
I have doubts about the name. This method only works for recycler views. But that's not very clear from the name.

I wonder if we should me more explicit, like `assertRecyclerItemDisplayedAtPosition()`. Or `assertDisplayedAtListPosition()`.
Or maybe I'm thinking too much and it's not really a problem. In the future we might improve it to make it compatible with ListViews too, like we did for `BaristaListInteractions`.

Opinions @SchibstedSpain/barista-mantainers?

Reply:
Agree with @Sloy 

Even more, adding a list id, position, view id and text is.... too much, is not easily to remember that method and the params

Maybe we can think on something more simplier, or try to reduce the number o params with helper class? not sure..

Reply:
`assertRecyclerItemDisplayedAtPosition` sounds good to me 👍

Reply:
`assertTextDisplayedAtPosition` sounds good? ???

Reply:
My proposal is
`assertListPositionHasText(recyclerId, position, String)`
`assertListPositionHasText(recyclerId, position, String, textViewId)`
`assertListPositionHasText(recyclerId, position, textRes)`
`assertListPositionHasText(recyclerId, position, textRes, textViewId)`

By the way, @DeanPike, please swear that you really need those methods 😅

Reply:
Hi Roc.

I do need this functionality as I`m using it in a project.